code/modules/hydroponics/grown/towercap.dm

```diff
@@ -140,6 +140,8 @@ obj/item/seeds/bamboo
 /obj/structure/punji_sticks/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/caltrop, 20, 30, 100, CALTROP_BYPASS_SHOES)
+	var/turf/T = get_turf(src)
+	usr.investigate_log("has placed punji sticks trap at [AREACOORD(T)].", INVESTIGATE_BOTANY)
```

Comment:
This should really be in the code that places the stick rather an in the sticks init. What about sticks created through game code / admin tools; this should work if they were to be needed.